assignments/this.js

```diff
@@ -32,6 +32,13 @@ console.log(oliveGarden.eat());
 
 // code example for New Binding
 
+function Food(favorites){
+    this.thingss = favorites;
+}
+let myFood = new Food("I love fruits");
+
+console.log(myFood.thingss);
+
 // Principle 4
 
 // code example for Explicit Binding
```

Comment:
Principle 4 needs more work. You need to use both call and apply. Also you might want to read the TK more on Explicit Binding.